[1m========================================================= test session starts =========================================================
platform linux -- Python 3.9.10, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /home/mariana/anaconda3/envs/mlflow-33d7d02cbcccb15d6d2d83802ac3094a9cc7388c/bin/python
cachedir: .pytest_cache
rootdir: /home/mariana/spotify-mlops/check_data
[1mcollected 2 items                                                                                                                     
test_data.py::test_column_presence_and_type [31mFAILED
test_data.py::test_class_names [31mFAILED
============================================================== FAILURES ===============================================================
[31m[1m____________________________________________________ test_column_presence_and_type ____________________________________________________
data =       danceability  energy  loudness  speechiness  acousticness  instrumentalness  liveness  valence  duration_ms genr...    -0.802       0.4290      0.018400          0.024300    0.0778   0.0974       164800  trap
[4461 rows x 10 columns]
    def test_column_presence_and_type(data):
        # Disregard the reference dataset
>       _, df = data
[31m[1mE       ValueError: too many values to unpack (expected 2)
[31m[1mtest_data.py[39m[22m:8: ValueError
[31m[1m__________________________________________________________ test_class_names ___________________________________________________________
data =       danceability  energy  loudness  speechiness  acousticness  instrumentalness  liveness  valence  duration_ms genr...    -0.802       0.4290      0.018400          0.024300    0.0778   0.0974       164800  trap
[4461 rows x 10 columns]
    def test_class_names(data):
        # Disregard the reference dataset
>       _, df = data
[31m[1mE       ValueError: too many values to unpack (expected 2)
[31m[1mtest_data.py[39m[22m:35: ValueError
======================================================= short test summary info =======================================================
FAILED test_data.py::test_column_presence_and_type - ValueError: too many values to unpack (expected 2)
FAILED test_data.py::test_class_names - ValueError: too many values to unpack (expected 2)
[31m========================================================= [1m2 failed[22m in 14.30s ==========================================================